docs/kubernetes.md

```diff
@@ -1,52 +1,23 @@
 # Faasm Kubernetes/ Knative integration
 
-Faasm is a runtime designed to be integrated into other serverless platforms.
```

Comment:
Simplified and rearranged this file. Moved all the cluster setup instructions to the bottom, and start with "Assuming you have a K8s cluster set up... if not see the bottom of this page."